synapse/tests/test_lib_certdir.py

```diff
@@ -39,7 +39,7 @@ def test_certdir_user(self):
     def test_certdir_host(self):
         with self.getCertDir() as cdir:
             cdir.genCaCert('syntest')
-            cdir.genHostCert('visi.vertex.link', signas='syntest')
+            cdir.genHostCert('visi.vertex.link', signas='syntest', sans='DNS:visi.vertex.link,DNS:vertex.link')
```

Comment:
I modified the server cert unit test to verify that the SANs were properly added